packages/clay-card/src/index.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import Context from './Context';
+
+import {AspectRatio} from './AspectRatio';
+import {Body, CardBodyProps} from './Body';
+import {Description} from './Description';
+import {Detail} from './Detail';
+import {Group} from './Group';
+
+type CardDisplayType = 'file' | 'image' | 'user';
+export interface CardProps {
+	displayType?: CardDisplayType;
+
+	/**
+	 * This property, when enabled, is used to apply horizontal styles into cards.
+	 */
+	horizontal?: boolean;
+
+	/**
+	 * This property, when enabled, is used to apply interactive styles into cards.
+	 */
+	interactive?: boolean;
+
+	/**
+	 * Flag that indicates if the card can be selectable.
+	 */
+	selectable?: boolean;
+}
+
+interface Props
+	extends CardProps,
+		React.BaseHTMLAttributes<HTMLAnchorElement | HTMLDivElement> {
+	children:
```

Comment:
Is it necessary to declare this `children` here? Since this can be customized, I think we do not have to force it to be a `CardBodyProps`.